src/test/java/com/conveyal/datatools/manager/jobs/MergeFeedsJobTest.java

```diff
@@ -98,6 +104,23 @@ public static void setUp() throws IOException {
             fakeAgency,
             zipFolderFiles("fake-agency-with-calendar-and-calendar-dates-3")
         );
+
+        // Other fake feeds for testing MTC MergeStrategy types.
+        FeedSource fakeTransit = new FeedSource("Fake Transit", project.id, MANUALLY_UPLOADED);
+        Persistence.feedSources.create(fakeTransit);
+        fakeTransitBase = createFeedVersion(fakeTransit, zipFolderFiles("merge-data-base"));
+        fakeTransitFuture = createFeedVersion(fakeTransit, zipFolderFiles("merge-data-future"));
+        fakeTransitModService = createFeedVersion(fakeTransit, zipFolderFiles("merge-data-mod-services"));
+        fakeTransitModTrips = createFeedVersion(fakeTransit, zipFolderFiles("merge-data-mod-trips"));
```

Comment:
I get what the first two feeds are, but what the last two feeds are is less clear.